apps/web/features/sample-mode/sample-mode-toggle.tsx

```diff
@@ -3,18 +3,20 @@
 import { Button } from "@repo/ui/components/button";
 import { FlaskConical, LogOut } from "lucide-react";
 import { useSampleMode } from "@/providers/sample-mode-provider";
+import { cn } from "@repo/ui/lib/utils";
 
 export default function SampleModeToggle() {
   const { isSampleMode, toggleSampleMode } = useSampleMode();
 
   return (
     <div className="pb-8 flex flex-col sm:flex-row items-start sm:items-center justify-between gap-2">
       <span
-        className={`text-sm px-3 py-1 rounded-md ${
+        className={cn(
+          "text-sm px-3 py-1 rounded-md font-medium",
           isSampleMode
-            ? "bg-amber-100 text-amber-800 font-medium"
-            : "text-neutral-600"
-        }`}
+            ? "bg-amber-100 text-amber-800 dark:bg-amber-900 dark:text-amber-200"
```

Comment:
check the clsx documentation, conditional classes could be managed in more convinient object way